src/core/Akka.API.Tests/CoreAPISpec.ApproveRemote.approved.txt

```diff
@@ -8,959 +8,6 @@
 [assembly: System.Runtime.InteropServices.ComVisibleAttribute(false)]
 [assembly: System.Runtime.InteropServices.GuidAttribute("78986bdb-73f7-4532-8e03-1c9ccbe8148e")]
 [assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.5", FrameworkDisplayName=".NET Framework 4.5")]
-public sealed class AckAndEnvelopeContainer : Google.ProtocolBuffers.GeneratedMessage<AckAndEnvelopeContainer, AckAndEnvelopeContainer.Builder>
```

Comment:
While I think this change is good, we need to check that this doesn't break the wire format with versions of Akka.NET 1.1.3 and earlier